Makefile

```
@@ -8,7 +8,7 @@ DIST_DIR       ?= $(CURDIR)/dist
 LEDGER_ENABLED ?= true
 TM_VERSION     := $(shell go list -m github.com/tendermint/tendermint | sed 's:.* ::')
 DOCKER         := $(shell which docker)
-PROJECT_NAME   = $(shell git remote get-url origin | xargs basename -s .git)
+PROJECT_NAME   = umee
```

```suggestion
PROJECT_NAME   := umee
```
- No need for a recursive assignment here ^^